app/kudos/models.py

```diff
@@ -646,6 +646,7 @@ class BulkTransferCoupon(SuperModel):
 
     sender_address = models.CharField(max_length=255, blank=True)
     sender_pk = models.CharField(max_length=255, blank=True)
+    quest_pk = models.CharField(max_length=255, blank=True)
```

Comment:
`models.ForeignKey` is probalby what u want to use not a charfield

Reply:
I did consider a foreignKey - but didn't think we would need to follow that association? I suppose it would future proof us though?

Reply:
no idea if we plan to follow the foreign key in the future, im just a fan of using the right type for the job.

Reply:
could we add in help_text as well while we add new fields ? 
I'm getting that done for existing fields

Reply:
@owocki - That's great, thanks for confirming.

@thelostone-mc - No worries, I'll add help_text with this next round of changes.